tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -0,0 +1,301 @@
+        => _ => _
+           .Executes(() =>
+           {
+               Logger.Info($"Prepare environment variables for debugger.");
+               //TODO TBD
+           })
+        ;
+
+    Target RunExplorationTestAssertions_Debugger
+        => _ => _
+               .Description("Run exploration test assertions.")
+               .Unlisted()
+               .Executes(() =>
+               {
+                   Logger.Info($"Running assertions tests for debugger.");
+                   //TODO TBD
+               });
+
+    Target RunExplorationTests_Profiler
+        => _ => _
+               .Description("Run exploration tests for profiler.")
+               .After(Clean)
+               .DependsOn(PrepareMonitoringEnvironment_Profiler)
+               .Executes(() =>
+               {
+                   GitCloneAndRunUnitTests(MonitoringType.Profiler);
+               })
+               .Triggers(RunExplorationTestAssertions_Profiler)
+        ;
+
+    Target PrepareMonitoringEnvironment_Profiler
+        => _ => _
+           .Executes(() =>
+           {
+               Logger.Info($"Prepare environment variables for profiler.");
+               //TODO TBD
+           })
+        ;
+
+    Target RunExplorationTestAssertions_Profiler
+        => _ => _
+               .Description("Run exploration test assertions.")
+               .Unlisted()
+               .Executes(() =>
+               {
+                   Logger.Info($"Running assertions tests for profiler.");
+                   //TODO TBD
+               });
+
+    void GitCloneAndRunUnitTests(MonitoringType monitoringType)
+    {
+        if (ExplorationTestName != null)
+        {
+            GitCloneAndRunUnitTest(ExplorationTestDescription.GetExplorationTestDescription(ExplorationTestName), monitoringType);
+        }
+        else
+        {
+            foreach (var testDescription in ExplorationTestDescription.GetAllTestDescriptions())
+            {
+                GitCloneAndRunUnitTest(testDescription, monitoringType);
+            }
+        }
+    }
+
+    void GitCloneAndRunUnitTest(ExplorationTestDescription testDescription, MonitoringType monitoringType)
+    {
+        if (Framework != null && !testDescription.IsFrameworkSupported(Framework))
+        {
+            Logger.Warn($"This framework '{Framework}' is not listed in the project's target frameworks of {testDescription.Name}");
+            return;
+        }
+
+        var projectPath = GiClone(testDescription);
+        if (!SkipBuild)
+        {
+            DotNetBuild(
+                x => x
+                    .SetProjectFile(projectPath)
+                    .SetConfiguration(BuildConfiguration)
+                    .SetTargetFramework(Framework)
+            );
+        }
+
+        DotNetTest(
+            x =>
+            {
+                x = x
+                   .SetProjectFile(projectPath)
+                   .SetNoRestore(true)
+                   .SetNoBuild(true)
```

Comment:
Analyzers will suggest you use these instead:
```suggestion
                   .EnableNoRestore()
                   .EnableNoBuild()
```